### DISCUSSION TITLE
Details on Zeptoforth's included features

### DISCUSSION THREAD

**Discussion by ctrilsbach** [tabemann/zeptoforth#15] [General]
Hi Travis,

Looking at the list of the many, many included features in Zeptoforth I have realised that I don't even know what of a lot of them mean ( Semaphores, message queues, action scheduler etc, etc) as they are at a much higher level than my Forth books. Is there a resource anywhere (book or weblink) that explains what these do. The reason for my question is that they might be useful for future projects but not knowing what they are I would not think to use them.

Thanks

Chris Trilsbach

**Comment by ctrilsbach** [comment]
Hi Travis,

Please ignore this discussion as I found the information in the Wiki page. Still getting used to Github.

Chris

**Comment by tabemann** [comment]
Oh, don't worry, any questions are good ones.
